implementation/leprechaun/src/line/controller/line.controller.ts

```diff
@@ -0,0 +1,67 @@
+import { Body, Controller, Delete, Get, HttpCode, Param, Patch, Post, Query } from '@nestjs/common';
+import { ApiCreatedResponse, ApiExtraModels, ApiNoContentResponse, ApiNotFoundResponse } from '@nestjs/swagger';
+import { ApiPaginatedResponse } from '../../shared/api/generic-paginated';
+import { PaginatedDto } from '../../shared/api/generic-paginated.dto';
+import { HttpExceptionDto } from '../../shared/api/http-exceptions';
+import { UUIDPipe, ValidateCreatePipe, ValidateUpdatePipe } from '../../shared/api/pipes';
+import { StopDto } from '../../stop/controller/stop.dto';
+import { LineService } from '../service/line.service';
+import { CreateLineDto, UpdateLineDto } from './line-create.dto';
+import { LineDto } from './line.dto';
+
+@Controller('/ext/v1/lines')
+@ApiExtraModels(StopDto)
+@ApiExtraModels(PaginatedDto)
+export class LineController {
+  constructor(private readonly lineService: LineService) {}
+
+  @Get('/')
+  @ApiPaginatedResponse(LineDto)
+  async getAllLines(
+    @Query('page') page_: number = 0,
+    @Query('size') size_: number = 50,
```

Comment:
![obraz](https://github.com/user-attachments/assets/87b5b670-7797-44ef-8b83-1da314adc280)
```suggestion
    @Query('size') size_: number = 10,
```